R/eval-walk.R

```diff
@@ -83,16 +87,18 @@ vars_select_eval <- function(vars,
   ensure_named(
     pos,
     vars,
-    uniquely_named,
-    allow_rename,
+    uniquely_named = uniquely_named,
+    allow_rename = allow_rename,
+    allow_empty = allow_empty,
     call = error_call
   )
 }
 
 ensure_named <- function(pos,
                          vars,
-                         uniquely_named,
-                         allow_rename,
+                         uniquely_named = FALSE,
+                         allow_rename = TRUE,
+                         allow_empty = TRUE,
                          call) {
```

Comment:
To improve the snapshot:

```suggestion
                         call = caller_env()) {
```